assets/js/settings/blocks/feature-flags.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
```

Comment:
One concern I have here is that we are now loading this api on the frontend for anything using block-settings right? is that problematic? I think I might have been the one who suggested putting this api in block-settings so that could be my fault here.

It doesn't look like it's affected build sizes significantly, so maybe this would be okay (since `@woocommerce/block-settings` is essentially just an alias for the path and thus it should be tree-shakeable).

Reply:
I had similar conerns, but I'm assuming this is tree shakeable, bundle bot didn’t report any extra weight so It’s safe.